src/Features/Core/Portable/SignatureHelp/SignatureHelpItem.cs

```diff
@@ -89,5 +89,14 @@ internal IEnumerable<TaggedText> GetAllParts()
                 Parameters.SelectMany(p => p.GetAllParts())).Concat(
                 DescriptionParts)));
         }
+
+        public override string ToString()
```

Comment:
Should this just call GetAllParts() above and string.Join that?

Reply:
➡️ No. I tried that to start with only to find that the handling of `SeparatorDisplayParts` and `Parameters` makes it unreadable.